MOBILE_API.xml

```diff
@@ -6775,12 +7013,8 @@
         
     </function>
     
-    <function name="SetDisplayLayout" functionID="SetDisplayLayoutID" messagetype="request" since="3.0">
-        <description>
-            Used to set an alternate display layout.
-            If not sent, default screen for given platform will be shown
-        </description>
-        
+    <function name="SetDisplayLayout" functionID="SetDisplayLayoutID" messagetype="request" deprecated="true" since="6.0">
+        <description>This RPC is deprecated. Use Show RPC to change layout.</description>
```

Comment:
@JackLivio should `history` be included for this deprecated function?

Reply:
Yes good catch. This should include the history tag.